Assets/Scripts/CameraController.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraController : MonoBehaviour
+{
+    [SerializeField]
```

Comment:
Let's move these attributes into a single line, like this:
`[SerializeField] private Camera mainCamera;`